yb-voyager/src/utils/utils.go

```diff
@@ -308,6 +310,49 @@ func CsvStringToSlice(str string) []string {
 	return result
 }
 
+func GetLocalIP() (string, error) {
+	if localIP != "" {
+		return localIP, nil
+	}
+
+	interfaces, err := net.Interfaces()
+	if err != nil {
+		return "", err
+	}
+	// Iterate through each network interface.
+	for _, iface := range interfaces {
```

Comment:
So, we're retrieving a valid IP from the first address from the first interface that we find? 
I don't know enough about this to comment if it's okay or not. Let's discuss this with Hemant/Nikhil?

Reply:
updated